cmd/launcher/launcher.go

```diff
@@ -198,6 +198,11 @@ func runLauncher(ctx context.Context, cancel func(), multiSlogger, systemMultiSl
 	flagController := flags.NewFlagController(slogger, stores[storage.AgentFlagsStore], fcOpts...)
 	k := knapsack.New(stores, flagController, db, multiSlogger, systemMultiSlogger)
 
+	// start counting uptime
+	processStartTime := time.Now()
+
+	k.UpTimeHistoryStore().Set([]byte("process_start_time"), []byte(processStartTime.Format(time.RFC3339)))
```

Comment:
`HistoryStore` makes me think this would hold a series of process start times - but I feel like we would want to do that eventually anyway so probably not worth changing the name. might be nice to add a comment somewhere (maybe here or near knapsack definition) saying that we only currently track the running process start time, but would like to push these into a historical list in the future

Reply:
Updated to LauncherHistoryStore